File: app/src/main/java/dev/ragnarok/fenrir/api/model/VKApiVideoAlbum.java
package dev.ragnarok.fenrir.api.model;

/**
 * Describes a photo album
 */
@SuppressWarnings("unused")
public class VKApiVideoAlbum {

    /**
     * Album ID.
     */
    public int id;

    /**
     * Album title.
     */
    public String title;

    /**
     * ID of the user or community that owns the album.
     */
    public int owner_id;

    public int count;

    /**
     * Date (in Unix time) the album was last updated.
     */
    public long updated_time;

    public String image;

    public VkApiPrivacy privacy;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiVideoAlbum.kt
package dev.ragnarok.fenrir.api.model

/**
 * Describes a photo album
 */
class VKApiVideoAlbum {
    /**
     * Album ID.
     */
    var id = 0

    /**
     * Album title.
     */
    var title: String? = null

    /**
     * ID of the user or community that owns the album.
     */
    var owner_id = 0
    var count = 0

    /**
     * Date (in Unix time) the album was last updated.
     */
    var updated_time: Long = 0
    var image: String? = null
    var privacy: VKApiPrivacy? = null
}